src/Navigation/WPF/Navigation.WPF/Service/NavigationService.cache.cs

```diff
@@ -21,25 +21,53 @@ public void ClearCache()
             ContentCache.Clear();
         }
 
-        private void CreateCachedContent(
-            Type viewType)
+
+        private static void CreateCachedContent(
+            INavigationRegistration registration)
         {
             if (!Configuration.CacheContent)
             {
                 return;
             }
 
-            if (Configuration.ContentTypesToExcludeFromCaching
-                .Contains(viewType))
+            if (Configuration.ContentTypesToExcludeFromCaching?
+                .Contains(registration.ViewType) == true)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5575746)